### PR TITLE
[chore] Resolve EIP feedback

### PR DESCRIPTION
**Pull request by Elliott-Green**
This resolves all of the comments made in the feedback on the EIP PR.
@DeFiFoFum I don't understand how the files are still being reverted. Need to make sure we're fetching the latest from master after merging.

### CODE REVIEW

File: EIP-draft.md [hunk 1]
```diff
@@ -262,7 +261,7 @@ The reference vesting NFT repository includes tests written in Hardhat.
 
 ## Reference Implementation
 
-A reference implementation of this EIP can be found in [eip-5725 assets](../assets/eip-5725/README.md/).
+A reference implementation of this EIP can be found in [eip-5725 assets](../assets/eip-5725/README).
```
Comment (DeFiFoFum) [inline EIP-draft.md:264]: Please leave `.md`. (ex: `../assets/eip-5725/README.md`), as the review was just mentioning the end `/`

**Review by DeFiFoFum — CHANGES_REQUESTED** [review]
Thanks @Elliott-Green for putting this together. 

I added one comment, please resolve that and then I will merge. 🙏

**Comment by Elliott-Green** [comment]
I notice that the IERC5725 function ordering change isn't present on the EIP fixes you've made.
If you run lint test it will fail because of ordering. Take the newer IERC5725 and the readme fixes.

**Comment by DeFiFoFum** [comment]
> I notice that the IERC5725 function ordering change isn't present on the EIP fixes you've made. If you run lint test it will fail because of ordering. Take the newer IERC5725 and the readme fixes.

I decided to follow a similar ordering pattern as ERC-721 and changed the ordering from `error` to `warn`.

**Review by DeFiFoFum — APPROVED** [review]

**Pull request merged**